src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -571,17 +571,13 @@ private static void CreateMethod(string propIdentifier, string propType, CodeCla
         
         Parallel.ForEach(unmappedRequestBuilderTypes, x => {
             var parentNS = x.Parent.Parent.Parent as CodeNamespace;
-            x.TypeDefinition = parentNS.FindChildrenByName<CodeClass>(x.Name)
-                                        .OrderBy(shortestNamespaceOrder)
-                                        .FirstOrDefault();
+            x.TypeDefinition = parentNS.FindChildrenByName<CodeClass>(x.Name).MinBy(shortestNamespaceOrder);
```

Comment:
I wasn't aware of this new MinBy method, really interesting, thanks for this update. We probably have many other cases which could be updated to minby.

Reply:
The generated code is compatible with dotnet standard 2.1 and c# 8 if I remember correctly and we intend to keep it this way for the near term future.